lib/support/requests/anonymous/anonymous_contact.rb

```diff
@@ -33,6 +30,11 @@ def requester
             only_actionable.
             order("created_at desc")
         }
+
+        private
+        def anonymous_contact_params
```

Comment:
Strong parameters do the user input filtering in the controller rather than in the model. As @benilovj notes, there are no controllers actually doing that work in this app, so nowhere for strong parameters `permit` calls to go.

We can definitely drop these methods – they're unused and if they were used would error due to `params` being undefined.